apps/docs/content/components/calendar/custom-cell-content.ts

```diff
@@ -0,0 +1,72 @@
+const App = `import {Calendar, CalendarCellContent, CalendarCellHeader, CalendarCellBody} from "@nextui-org/react";
```

Comment:
please refer to other files under `http://apps/docs/content/components/`. this doc format was outdated.